shell/assets/translations/ru-ru.yaml

```
@@ -531,3 +531,769 @@ authConfig:
+          so {Хранилище: }
+          standard {Стандартный: }
+          other {}
+        }{memoryGb} GB, {vcpus, plural,
+          =1 {# vCPU}
+          other {# vCPUs}
+        }, {disk} GB диск ({value})
+      tags:
+        label: Droplet теги
+        placeholder: например, my_server
+    linode:
+      typeLabel: |-
+        {label}, {vcpus, plural,
+          =1 {# vCPU}
+          other {# vCPUs}
+        }, {disk} GB диск ({value})
+    vsphere:
+      hostOptions:
+        any: Любой
+      vAppOptions:
+        label: Параметры vApp 
+        description: Выберите свойства окружения OVF
+        disable: Не использовать vApp
+        auto: Использовать vApp для конфигурации сети с сетевыми профилями протоколов
+        manual: Предоставить пользовательскую конфигурацию vApp
+        restoreType: Восстановить тип
+        transport:
+          label: Транспорт окружения OVF
+          tooltip: com.vmware.guestInfo or iso
+          placeholder: например, com.vmware.guestInfo
+        protocol:
+          label: IP-протокол vApp 
+          tooltip: IPv4 или IPv6
+          placeholder: например, IPv4
+        allocation:
+          label: Политика распределения IP-адресов vApp 
+          tooltip: dhcp, fixed, transient or fixedAllocated
+          placeholder: например, fixedAllocated
+        properties:
+          label: свойства vApp
+          add: Добавить свойство
+          keyPlaceholder: например, guestinfo.interface.0.ip.0.address
+          valuePlaceholder: например, ip:VM Network, expression or string
+      networks:
+        label: Сети
+        add: Добавить сеть
+      guestinfo:
+        label: Параметры конфигурации, используемые для guestinfo
+        add: Добавить параметр
+        keyPlaceholder: например, guestinfo.hostname
+        valuePlaceholder: например, myrancherhost
+      creationMethods:
+        template: 'Развернуть из шаблона: Дата-центр'
+        library: 'Развернуть из шаблона: Библиотека контента'
+        vm: 'Клонировать существующую виртуальную машину'
+        legacy: 'Установить из boot2docker ISO (устаревшее)'
+      scheduling:
+        label: Планирование
+        description: Выберите гипервизор, на который будет назначена виртуальная машина
+        dataCenter: Дата-центр
+        resourcePool: Пул ресурсов
+        dataStore: Хранилище данных
+        folder: Папка
+        host:
+          label: Хост
+          note: Хост для создания ВМ (оставььте пустым для автономного ESXi или кластера с DRS)
+      instanceOptions:
+        label: Свойства экземпляра
+        description: Выберите размер и ОС виртуальной машины
+        cpus: CPUs
+        memory: Память
+        disk: Диск
+        creationMethod: Метод создания
+        template:
+          label: Шаблон
+          none: Шаблоны не найдены
+        contentLibrary: Библиотека контента
+        libraryTemplate: Библиотека шаблонов
+        virtualMachine: Виртуальная машина
+        osIsoUrl:
+          label: ОС ISO URL
```

```suggestion
          label: URL до ISO ОС
```
Я думаю что тут имеется в виду ссылка iso образа операционной системы